cmd/main.go

```diff
@@ -112,6 +114,7 @@ type app struct {
 	cache    cache.Cache
 	limiter  limiter.RateLimiter
 	searcher searcher.Searcher
+	fflag    fflag.FeatureFlag
```

Comment:
@Dotunj We don't need this DI from the `cmd` package. From my understanding, this is only an HTTP Client. We should instantiate it where it's needed and call the API.